packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -210,7 +211,8 @@
                 "hoverReference",
                 "upvote",
                 "downvote",
-                "clickBodyLink"
+                "clickBodyLink",
+                "viewDiff"
```

Comment:
please upstream this: https://github.com/aws/aws-toolkit-common/blob/df10c59a61083162b6d97d2d4854a47debb01465/telemetry/definitions/commonDefinitions.json#L982